docs/DeveloperGuide.adoc

```diff
@@ -230,6 +230,119 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+//tag::SortCommand[]
+=== SortCommand feature
+==== Current Implementation
+The sort command feature allows the sales person to sort their clients according to their names in alphebetical order. This will allow for a more organized and neat address book.
+
+To sort the adress book, simply type 'sort' or the command alias 'so'.
+
+An example of the sort command would be: sort
+
+An example of the sort command alias would be: so
+//end::SortCommand[]
+
+// tag::DeletTag[]
+=== DeletTag feature
+==== Current Implementation
+The delete tag command allows user to remove a particular tag from all persons in the AddressBook. For example, he may want to delete the tag "neigbour" from all his contacts if he is no longer staying at the place.
+
+We implement Model#deleteTag(Tag) torwards the direction of removeTag command, so that the developer dealing with the 'logic' component is able to implement such a command.
+
+==== Design Considerations
+
+===== Aspect: Implementation of time zone
+
+The current codebase has a flaw in tags management. Tags no longer in use by anyone may still exist on the AddressBook. This may cause some tests to fail.
```

Comment:
Why are you talking about tags in Time zone implementation? I am a bit confused 😛